exercise/templates/exercise/exercise_base.html

```diff
@@ -30,6 +30,10 @@
 {% endblock %}
 {% endblock %}
 
+{% block exercise_title %}
+<h3 class = "exercise_title">{{ current.name }}</h3>
```

Comment:
There shoudn't  be a space between the attribute name, equals sign and the value.